README.md

```diff
@@ -11,49 +11,171 @@
+|   force	        |   `boolean`	| Whether or not to force push the reference even if the ancestor commits differs. Default is `false`. |
 
 
-* [Versioning](#versioning)
-* [Contributing](#contributing)
-* [License](#license)
 
-## Quickstart
+#### `logger`
+*[Logger](https://www.npmjs.com/package/@types/pino)* <br>
+The default logger is [Pino](https://github.com/pinojs/pino). You can plug in any logger that conforms to [Pino's interface](https://www.npmjs.com/package/@types/pino)
 
-### Installing the client library
+
+#### Exceptions
+The core-library will throw an exception if the [GitHub V3 API](https://developer.github.com/v3/) returns an error response, or if the response data format did not come back as expected. <br>
+
+## CLI
+
+### Installation
 
 ```bash
-npm install @google-cloud/code-suggester
+npm i code-suggester -g
 ```
 
+### code-suggester pr
+`code-suggester pr` - opens a GitHub Pull Request against the upstream primary branch with the provided set of changes.
+
+#### Syntax
+
+
+`code-suggester pr [options] --upstream-repo=<string> --upstream-owner=<string>`
+
+
+#### Options
 
+#### `--upstream-repo, -r`
+*string* <br>
+**Required.** The repository to create the fork off of.
 
 
+#### `--upstream-owner, -o`
+*string* <br>
+**Required.** The owner of the upstream repository.
+
+
+#### `--description, -d`
+*string* <br>
+The GitHub Pull Request description. Default value is: `'code suggestions'`.
+
+#### `--title, -t`
+*string* <br>
+The GitHub Pull Request title. Default value is: `'chore: code suggestions'`.
+
+#### `--branch, -b`
+*string* <br>
+The GitHub working branch name. Default value is: `'code-suggestions'`.
+
+#### `--primary, -p`
+*string* <br>
+The primary upstream branch to open a PR against. Default value is: `'master'`.
+
+#### `--message, -m`
+*string* <br>
+The GitHub commit message. Default value is: `'code suggestions'`.
+
+#### `--force, -f`
+*boolean* <br>
+Whether or not to force push a reference with different commit history before the remote reference HEAD. Default value is: `false`.
+
+#### `--files [<file1>...]`
+*string* <br>
+**Required.** A list of files
+
+**Note:** Use either `--files` or `--git-dir` exclusively. Using both with terminate with an error.
+
+#### `--git-dir`
+*string* <br>
+**Required.** The path of a git directory
+
+**Note:** Use either `--files` or `--git-dir` exclusively. Using both with terminate with an error.
+
+
+### Example
+```
+code-suggester pr -o 'Foo' -r 'Bar' --git-dir="/my-project"
+```
 
 ## Supported Node.js Versions
 
 Our client libraries follow the [Node.js release schedule](https://nodejs.org/en/about/releases/).
 Libraries are compatible with all current _active_ and _maintenance_ versions of
 Node.js.
 
-Client libraries targetting some end-of-life versions of Node.js are available, and
+Client libraries targeting some end-of-life versions of Node.js are available, and
```

Comment:
Why do I have a hunch I'm going to find this typo in 94 other node.js repositories?  Could I trouble you to check 😆